bus-mapping/src/circuit_input_builder/execution.rs

```diff
@@ -71,6 +71,18 @@ impl ExecStep {
             error: None,
         }
     }
+
+    /// Returns `true` if `error` is oog and stack related..
+    pub fn oog_or_stack_error(&self) -> bool {
+        if self.error.is_none() {
+            return false;
+        }
+
+        matches!(
+            self.error.clone().unwrap(),
+            ExecError::OutOfGas(_) | ExecError::StackOverflow | ExecError::StackUnderflow
+        )
```

Comment:
Looks like it can be simplified
```suggestion
        matches!(self.error, Some(ExecError::OutOfGas(_) | ExecError::StackOverflow | ExecError::StackUnderflow))
```

Reply:
updated in [aa107c0](https://github.com/privacy-scaling-explorations/zkevm-circuits/pull/674/commits/aa107c0271feb0aa70118b0d596dcb34304d9afd) thanks !!